src/Uno.UWP/ApplicationModel/Contacts/ContactAddress.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactAddress
+	{
+		public string StreetAddress { get; set; }
+		public string Region { get; set; }
+		public string PostalCode { get; set; }
+		public string Locality { get; set; }
+		public ContactAddressKind Kind { get; set; }
+		// public string Description { get; set; }
+		public string Country { get; set; }
+		public ContactAddress()

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4900834)

Reply:
What is better? Adding empty constructor to implementation, or adding lines to diffignore?

Reply:
An empty construction like this is already generated by the compilator when there's no other constructor defined. It's called a "default constructor".

So defining a default constructor with nothing in it is useless and is triggering this analysis issue.

Reply:
And when I remove this constructor, diffignore would not show error that ctor is missing?